packages/odyssey-react/package.json

```diff
@@ -40,6 +43,7 @@
     "build:clean": "rm -rf dist",
     "build:types": "tsc --project tsconfig.production.json",
     "build:source": "NODE_ENV=production babel src --out-dir dist --source-maps --extensions '.js,.ts,.jsx,.tsx,.module.scss'",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "build-icons": "svgr $(node resolveIconSrcPath.js) --out-dir ./src/components/Icon/"
```

Comment:
consider moving this out of the root of the package and into a dedicated `scripts` directory. [This is the approach I'm using here](https://github.com/okta/odyssey/pull/1057/files#diff-c2f635874746aee7145901da097d8cb119a701d24f79b7b53df39edad84d617a), and is a conventional pattern at Okta

Reply:
Sadly, running resolution with node -p from within package.json fails to resolve correctly